packages/lib/modules/pool/actions/add-liquidity/AddLiquidityProvider.tsx

```diff
@@ -151,6 +151,7 @@ export function _useAddLiquidity(urlTxHash?: Hash) {
     [!isConnected, LABELS.walletNotConnected],
     [areEmptyAmounts(humanAmountsIn), 'You must specify one or more token amounts'],
     [hasValidationErrors, 'Errors in token inputs'],
+    [needsToUpdateInputs, 'Update inputs first'],
```

Comment:
I don't think we should be introducing this as a new state variable. We already have functionality for handling validation type errors. Please try to use the functionality from `TokenInputsValidationProvider` instead. This should then result in the preview being blocked by the `disabledConditions` in the `AddLiquidityProvider`.

Reply:
This has been removed.